src/test/controllers/users.signout.test.js

```diff
@@ -7,17 +7,14 @@ describe('Test User API', function () {
       rp({
         method: 'GET',
         uri: 'http://localhost:8000/users/signOut',
-        jar: true,
         resolveWithFullResponse: true,
         json: true,
       })
         .then(function (result) {
-          result.statusCode.should.equal(200);
-          result.body.msg.should.equal(userCallback.SUCCESS_SIGNOUT);
+          result.statusCode.should(200);
```

Comment:
Yup. Now /signout is unneccesary. However we can make it as stamp time when user is got out : )